Tests/SwiftParserTest/Expressions.swift

```diff
@@ -354,10 +360,10 @@ final class ExpressionTests: XCTestCase {
 
     AssertParse(
       ##"""
-      #"#^DIAG^#
+      #^NOTE^##"#^DIAG^#
```

Comment:
Not used any more?

Reply:
https://github.com/apple/swift-syntax/pull/878